app.rb

```diff
@@ -4,9 +4,11 @@
 require './book'
 require './classroom'
 require './rental'
+require './print_script'
 
-class App
+class App < PrintScript
```

Comment:
The SOLID could be improved,

Please, kindly:

- [ ] The methods `create_person`, `create_student`, `create_book` and `create_rental` should be outside from this class.  That way, this class will be only for listing things (e.g. "list_all_rentals").  Please, kindly do so.
- [ ] All the requests for input and validation should be moved to `print_script.rb`, on your class `PrintScript` where your interface lives.  In this case, all the interface and user interactions should live in that class.
- [ ] The line `student = Student.new(age, 'classroom', name, parent_permission: false)` should be placed in the `create_people` method (students are people).
- [ ] Also, the line `teacher = Teacher.new(age, specialization, name)` should be placed in the `create_people` method as well (teachers are people).
- [ ] The `start_console` method which runs the main interface, should be out of the `App` class (this class is just a lister).  

## Explanation

Every class should have a single responsibility and methods that relate to the main functionality of that class.  Also, the user input, the "puts", and the interface should be apart from the logic.  

## Hints

- All requests for input and input validation should be placed in your `print_script.rb` file.  That way, we comply with the single responsibility rule.

- Also, in this case, the `app.rb`, should just serve as a "lister" to populate the screen with data that the user requests (like `list_all_rentals`, etc).

- The `start_console` method could be better put in the `print_script` class.

That way, if you add more functionality in the future, you won't need to refactor the whole thing, thus making the app more easily scalable. 💪

Reply:
Hello @Meltrust  Thanks for the review, the changes are made.